src/Moryx.Products.Model/Migrations/ProductsContextModelSnapshot.cs

```diff
@@ -685,6 +685,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<long>("WorkplanId")
                         .HasColumnType("bigint");
 
+                    b.Property<long?>("PositionX")
+                        .HasColumnType("bigint");
+
+                    b.Property<long?>("PositionY")
```

Comment:
I think for people comming from the last MORYX version we need to update the SQL scripts here for the DB migration 🤔🙈